presto-main/src/test/java/io/prestosql/sql/planner/TestDynamicFilter.java

```diff
@@ -44,10 +48,13 @@
 {
     private final Metadata metadata = createTestMetadataManager();
 
-    TestDynamicFilter()
+    public TestDynamicFilter()
     {
         // in order to test testUncorrelatedSubqueries with Dynamic Filtering, enable it
-        super(ImmutableMap.of(ENABLE_DYNAMIC_FILTERING, "true"));
+        super(ImmutableMap.of(
+                ENABLE_DYNAMIC_FILTERING, "true",
+                JOIN_REORDERING_STRATEGY, JoinReorderingStrategy.NONE.name(),
+                JOIN_DISTRIBUTION_TYPE, JoinDistributionType.BROADCAST.name()));
```

Comment:
I don't think:
```
JOIN_DISTRIBUTION_TYPE, JoinDistributionType.BROADCAST.name())
```
is needed. Could you try remove it?

Reply:
Sure. Removed.I've removed it along with JOIN_REORDERING_STRATEGY moving it to a Session used in test-cases when needed.